index.d.ts

```diff
@@ -2807,7 +2799,7 @@ declare namespace Shopify {
     payment_details: IPaymentDetails;
     parent_id: number;
     processed_at: string;
-    receipt: ITRansactionReceipt;
+    receipt: Record<string, unknown>;
```

Comment:
I think this is not backwards compatible with TypeScript 2.

Reply:
`Record` was introduced in TypeScript 2.1: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-1.html#partial-readonly-record-and-pick

Reply:
Yes but AFAIK `unknown` was introduced in TypeScript

Reply:
Ah, you’re right. Changed to `any`